### PR TITLE
Manuals rewriting for exceptions & troubleshooting

### PR DESCRIPTION
**Pull request by ttodua**
Have been trying to rewrite manuals to be compact, better browsable and not with repeated/duplicate information, as too much stretched explanations could be distracting for users. Better to be compact informative.

### CODE REVIEW

File: wiki/Manual.md [hunk 1]
```diff
@@ -5953,109 +5953,26 @@ class BaseError (Exception):
 class BaseError extends \Exception {}
 ```
 
-Below is an outline of exception inheritance hierarchy:
+Here is an outline of exception inheritance hierarchy: https://github.com/ccxt/ccxt/blob/master/ts/src/base/errorHierarchy.ts
 
-```text
```
Comment (ttodua) [inline wiki/Manual.md:5958]: we'd better get rid of such ASCII drawing, which takes much space and it's even being manually maintained here. make users to look into the self-explanatory JSON tree, anyone would be able to read there.

File: wiki/Manual.md [hunk 2]
```diff
@@ -5953,109 +5953,26 @@ class BaseError (Exception):
 class BaseError extends \Exception {}
 ```
 
-Below is an outline of exception inheritance hierarchy:
+Here is an outline of exception inheritance hierarchy: https://github.com/ccxt/ccxt/blob/master/ts/src/base/errorHierarchy.ts
```
Comment (carlosmiei) [inline wiki/Manual.md:5956]: @ttodua let's keep the chart here, it's easier to see how the exceptions are organize without having to open a new link

File: wiki/Manual.md (hunk 1 above)
Comment (ttodua) [reply wiki/Manual.md:5958]: but if you look at it closely, it's harder to read (with some plus and half-cut lines) than JSON...

Comment (ttodua) [reply wiki/Manual.md:5958]: well.. as you say,ok.

File: wiki/Manual.md (hunk 2 above)
Comment (ttodua) [reply wiki/Manual.md:5956]: restored

**Comment by ttodua** [comment]
@carlosmiei I have still some thoughts whether `OperationFailed` should be under `ExchangeError` or `NetworkError` ?

**Comment by ttodua** [comment]
but in the manuals, I have found out recently, that we had such explanation that "NetworkErrors are the ones that can be retried, as they are of 'temporary' nature", while `OperationFailed` is also same.  If you look closely under `NetworkError`, it's not actually network related only, but "RateLimitExceeded" (which is response from API ENGINE itself, so it is nowhere like `network-error`) or say `InvalidNonce` is also under NetworkError. also `ExchangeNotAvailable` is under NetworkError.

That makes me think that `OperationFailed` (which is similarly temporary nature, alike "exchange-not-available") seems much familiar to that group rather than Request side errors (under ExchangeError).

please give it a second thought and lmk.

File: wiki/Manual.md [hunk 3]
```diff
@@ -6138,7 +6140,7 @@ In case you experience any difficulty connecting to a particular exchange, do th
 - Check if there were any news from the exchange recently regarding downtime for maintenance. Some exchanges go offline for updates regularly (like once a week).
 - Make sure that your system time in sync with the rest of the world's clocks since otherwise you may get invalid nonce errors.
 
-## Notes
+Further notes:
```
Comment (carlosmiei) [inline wiki/Manual.md:6143]: I think we should highlight this title otherwise it's hard to see

Comment (ttodua) [reply wiki/Manual.md:6143]: I wanted to de-hyperlink that (so it was not hashtaged under `#notes` url), would you might to edit this file on the fly and commit per you think it to be better? i.e. bolded, or reverted back into `##` title?

**Comment by frosty00** [comment]
> That makes me think that OperationFailed (which is similarly temporary nature, alike "exchange-not-available") seems much related/similar to that group rather than Request side errors (under ExchangeError).

Yeah we could put it underneath NetworkError I think that would be ok

**Comment by frosty00** [comment]
if you can add that edit will merge this

**Comment by ttodua** [comment]
@frosty00 ok, pushed, lmk if that's ok

**Pull request merged**